server_vllm.py

```diff
@@ -306,11 +347,25 @@ async def wrap_vllm_generator() -> AsyncGenerator[Tuple[str, Optional[str]], Non
                     delta_text.strip()
                     not in prompt_template.get_stop_tokens_for_generation()
                 ):
+                    if (
+                        previous_texts == delta_text
```

Comment:
This is equivalent to: len(previous_texts) == 0, right ? can you add comment:  "At first, yield the prefix based on tool_choice"